packages/website/content/docs.md

```diff
@@ -324,6 +324,7 @@ The `setting` action defines a user-configurable option that can be referenced i
 **Optional Inputs**
 
 - `defaultValue` - Specifies the value to be used by this setting if the user doesn’t provide a value at run time.
+- `description` - Describe this setting for help information.
```

Comment:
maybe say "Human friendly explanation of the setting's affect on the built chain"